src/docfx/lib/schema/JsonSchemaValidator.cs

```diff
@@ -253,6 +254,32 @@ private void ValidateDeprecated(JsonSchema schema,  JToken token, List<Error> er
             }
         }
 
+        private void ValidateDependentEnums(JsonSchema schema, JObject map, List<Error> errors)
+        {
+            foreach (var fieldName in schema.DependentEnums.Keys)
```

Comment:
can use tuple deconstruction here to save another dictionary lookup:
```suggestion
            foreach (var (fieldName, (fieldValues) in schema.DependentEnums.Keys)
```

Reply:
sure